initialize.sh

```diff
@@ -2,45 +2,93 @@
 
 set -e
 
-if [[ -f "./.stellar/contract-ids/account.json" ]]; then
-  echo "Found existing './.stellar' directory; already initialized."
-  exit 0
-fi
+NETWORK="testnet"
+RPC_URL="https://soroban-testnet.stellar.org"
+NETWORK_PASSPHRASE="Test SDF Network ; September 2015"
```

Comment:
New stellar-cli already knows all the passphrases for predefined networks. I removed them consciously

Reply:
Alright